src/hyperlight_host/src/hypervisor/mod.rs

```diff
@@ -303,6 +303,9 @@ pub(crate) trait Hypervisor: Debug + Send {
     #[cfg(crashdump)]
     fn crashdump_context(&self) -> Result<Option<crashdump::CrashDumpContext>>;
 
+    #[cfg(crashdump)]
+    fn runtime_config(&self) -> &crate::sandbox::uninitialized::SandboxRuntimeConfig;
```

Comment:
I don't think we should add this method to the Vm because it's not conceptually related to vms. It's already part of uninitialized sandbox so I believe you can already grab it from there without going through here